examples/baremetal/values.yaml

```diff
@@ -0,0 +1,47 @@
+# local-config: referenced, but not emitted by kustomize
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edpm-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  preProvisioned: false
+  nodeset:
+    baremetalsettemplate:
+      bmhLabelSelector:
+        app: openstack
+      ctlplaneInterface: enp1s0
+      cloudUserName: cloud-admin
+    ansible:
+      vars:
+        # edpm firewall, change the allowed CIDR if needed
+        edpm_sshd_allowed_ranges: ['192.168.111.0/24']
+    networks:
```

Comment:
Would not this create more issues and confusion as values.yaml has networks as a subkey in nodeset where as it maps to spec.nodeTemplate.networks?